### PR TITLE
Added recipe for Flask-Creole

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/flask-creole/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - flaskext
+    - flaskext.creole
+
+about:
+  home: http://bitbucket.org/aafshar/flask-creole-main
+  license: MIT
+  # license_file: No LICENSE in MANIFEST.in
```
Comment (jakirkham) [inline recipes/flask-creole/meta.yaml]: Please raise an issue/PR upstream to get this fixed.

File: recipes/flask-creole/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  skip: True  # [py3k and py>33]
```
Comment (jakirkham) [inline recipes/flask-creole/meta.yaml]: 😕 Wouldn't that be the same as just `# [py3k]`?

File: recipes/flask-creole/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/flask-creole/meta.yaml]: See https://bitbucket.org/aafshar/flask-creole-main/pull-requests/1/

**Comment by jakirkham** [comment]
Toggling to re-run CIs.

Note to self: The linter correctly handle this case and did not raise a merge conflict thanks to PR ( https://github.com/conda-forge/conda-forge-webservices/pull/68 ).

**Pull request merged**